casual/casual-api/src/main/java/se/laz/casual/config/Inbound.java

```diff
@@ -13,7 +13,7 @@
 {
     public static final String CASUAL_INBOUND_STARTUP_MODE = "CASUAL_INBOUND_STARTUP_MODE";
     public static final String CASUAL_INBOUND_USE_EPOLL = "CASUAL_INBOUND_USE_EPOLL";
-    public static final String CASUAL_INBOUND_INITIAL_DELAY_ENV_NAME = "CASUAL_INBOUND_INITIAL_DELAY";
+    public static final String CASUAL_INBOUND_STARTUP_INITIAL_DELAY_ENV_NAME = "CASUAL_INBOUND_STARTUP_INITIAL_DELAY_IN_SECONDS";
```

Comment:
`_IN_SECONDS` feels OTT. What do you think @TobiasLeo 
Eitherway consistency in the const naming would be nice.

Reply:
@TobiasLeo Think of a person, not the one who configured it, who reads the env var and the value.
With IN_SECONDS, they know what it is - without it, they'd have to guess or go read some other documentation.
It is a bit long but I think the information conveyed makes it well worth it, less error prone all around.

Reply:
I think it is useful to have  the unit stated in the env if it is fixed. Any user stumbling upon it will understand the value more easily without having to look at the documentation or source code. Personally I would drop the IN though as I feel it is kind of redundant. I often feel like having <some time related task>_<unit> to be sufficient to tell the user what is configured and what the number will be interpreted as

I agree with @cklellie about naming consistency, either they should all be suffixed with _ENV_NAME or none of them should have the suffix

Reply:
I'll drop the IN part.
Re ENV_NAME for constants that are names for env vars, if you check the rest of the code base it is with ENV_NAME since that conveys the meaning of that the constant is.
Thus. the other ones, if they are indeed env vars, should be changed - not in this PR though.
I'll create an issue to go over ALL constants in the code and make sure that env vars do have ENV_NAME in them ( I think 99% do already have that).

Reply:
https://github.com/casualcore/casual-java/issues/54